app/controllers/users_controller.rb

```diff
@@ -0,0 +1,23 @@
+class UsersController < ApplicationController
+  def new
+    redirect_to home_path unless !logged_in?
```

Comment:
`unless !` is very confusing, it is just `if`. And `if !` is just `unless` - I do not recommend overusing `!`